### PR TITLE
syncing used instead of saving while logged in

### PR DESCRIPTION
**Pull request by himanish-star**
fixes : #597 

Changes can be seen in the GIF below
ezgif com-crop

### CODE REVIEW

File: locales/en-US/notes.properties [hunk 1]
```diff
@@ -17,7 +17,7 @@ disableSync=Disable Sync
 reconnectSync=Reconnect to Sync
 syncError=Sync Error
 
-savingChanges=Saving changes…
+savingChanges=Syncing…
```
Comment (flodolo) [inline locales/en-US/notes.properties]: If you change a string you also need to change the associated ID
https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Changing_existing_strings

Also make sure to update the reference in comments (if available)

**Comment by Natim** [comment]
This is not the proper way of fixing the issue. You can't change the property file like that.

**Comment by himanish-star** [comment]
@Natim , I was about fix this. Could you reopen this PR ? I'll try this another way.

**Comment by himanish-star** [comment]
This is with respect to https://github.com/mozilla/notes/issues/597#issuecomment-358702806

> After a note has been edited, footer should say "Syncing…"

I tried searching for the word "Syncing" but I found out "Syncing changes". Thus I have used this as you have told me that I can't edit the property file like that.
> Before a note has been edited it should say "Last Synced HH:MM"

Same for this I couldn't find "Last Synced" so I did it with "Synced at" instead.
> It does this but also has Save and Saving…

So do you want me to remove this. For now I have removed this , if you want it back then I'll fix this back too :smiley:
Here's the new GIF
simplescreenrecorder-2018-01-22_16 56 02
__________________________
If this is not what you want, then could you explain me a bit more :smiley:

**Comment by himanish-star** [comment]
@Natim , any views on the recent changes ?

**Comment by Natim** [comment]
If we need to go back to _Synced 7:24 PM_ it is a regression of https://github.com/mozilla/notes/pull/467 and @ryanfeeley @vladikoff and @flodolo need to talk to figure out what made that change that I don't remember.

**Comment by Natim** [comment]
Writting the following made me understand why we added the at inside:

```
# LOCALIZATION NOTE (syncComplete): {time} is the time of last sync. If this
# structure doesn't work for your locale, you can translate this as "Synced at {time}".
syncComplete3=Synced {time}
```

@ryanfeeley is that really what we want?

**Comment by ryanfeeley** [comment]
@Natim Yes, an example like *Synced 7:24 PM* is perfect. Same for *Saved 7:24 PM*.

**Comment by himanish-star** [comment]
ezgif com-video-to-gif
This is a new GIF. Please see the changes

**Comment by ryanfeeley** [comment]
@himanish-star Is it putting the new text at the end (where the cursor is) or the beginning? Hard to tell as it's the same text.

**Comment by himanish-star** [comment]
Well it's putting the text according to the previous caret position. There is an issue already on this #532

File: src/sidebar/panel.js [hunk 2]
```diff
@@ -122,7 +122,9 @@ ClassicEditor.create(document.querySelector('#editor'), {
               setAnimation(true); // animateSyncIcon, syncingLayout, warning
               syncingInProcess = true;
             }
-            if (! waitingToReconnect) {
+            if (! waitingToReconnect && isAuthenticated) {
```
Comment (Natim) [inline src/sidebar/panel.js]: I guess you can have a if for the `!waitingToReconnect` and another one for the `isAuthenticated` condition.

Comment (himanish-star) [reply src/sidebar/panel.js]: Oh, I just saw this now. Well thanks for changing :smiley:

**Pull request merged**